core/test/financial-templates/Liquidatable.js

```diff
@@ -361,14 +361,16 @@ contract("Liquidatable", function(accounts) {
         unreachableDeadline,
         { from: liquidator }
       );
+      const liquidationTime = await liquidationContract.getCurrentTime();
```

Comment:
Instead of querying the contract for its current time, I would grab the block-timestamp from the `createLiquidationResult`. Even though for this test they should be the same, it seems slightly safer to check that the function block timestamp is sending in the event.

Reply:
im noticing a wierd issue. when using `getCurrentTime()` within the contract, i get a different timestamp vs the block the transaction is mined in, this causes test to fail. when emitting the event using `now` instead, the test passes.

Reply:
@nicholaspai just chatted a bit with @daywiss and we arrived at the conclusion that the right way to go is to leave the code as it is.

The reason testing block time is failing is because we use simulated times in tests, so the block time won't line up with `getCurrentTime()` unless you force ganache to mine the block at the same time.

Reply:
Makes sense I overlooked this.